pytest_asdf/plugin.py

```diff
@@ -59,8 +59,8 @@ def from_parent(
         fspath,
         skip_examples=False,
         validate_default=True,
-        ignore_unrecognized_tag=False,
-        ignore_version_mismatch=False,
```

Comment:
This doesn't match the default so switching to None will change this behavior:
https://github.com/asdf-format/asdf/blob/b8e1759aa9f013283942140c94732a170eb9a703/asdf/asdf.py#L1772
I'm not immediately understanding the context so does this change matter?